packages/cli/src/commands/build.js

```diff
@@ -64,6 +65,7 @@ export const builder = (yargs) => {
     .option('esbuild', {
       type: 'boolean',
       required: false,
+      default: getConfig().experimental.esbuild,
```

Comment:
This will run when the command is imported. I think `yargs` allows the default value to be a function, and then it's only evaluated when the command is run.

I think this might work:
```suggestion
      default() { return getConfig().experimental.esbuild },
```

Reply:
I just tested this, it doesn't seem that a default function can be supplied.